services/src/northbound/src/main/java/org/openkilda/northbound/controller/LinkController.java

```diff
@@ -150,4 +150,18 @@
                                                                 @RequestParam(value = "dst_port") Integer dstPort) {
         return linkService.rerouteFlowsForLink(srcSwitch, srcPort, dstSwitch, dstPort);
     }
+
+    /**
+     * Update "Under maintenance" flag in the link.
+     *
+     * @return updated link.
+     */
+    @ApiOperation(value = "Update \"Under maintenance\" flag in the link.", response = LinkDto.class,
+            responseContainer = "LinkDto")
+    @PatchMapping(path = "/links/under-maintenance",
+            produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    @ResponseStatus(HttpStatus.OK)
+    public CompletableFuture<List<LinkDto>> updateIslUnderMaintenance(@RequestBody LinkDto link) {
```

Comment:
Why do you pass the src/dst switch and port in the request body, although in another cases (https://github.com/telstra/open-kilda/pull/1346 and https://github.com/telstra/open-kilda/pull/1724) query parameters are used. See https://tools.ietf.org/html/rfc3986#section-3.4